core/runtime.rs

```diff
@@ -157,6 +156,8 @@ pub(crate) struct ContextState {
   pub(crate) js_wasm_streaming_cb: Option<v8::Global<v8::Function>>,
   pub(crate) pending_promise_rejections:
     HashMap<v8::Global<v8::Promise>, v8::Global<v8::Value>>,
+  pending_dyn_mod_evaluate: Vec<DynImportModEvaluate>,
```

Comment:
Why make it public?

Reply:
This change removes the `pub(crate)` visibility specifier, which makes the field private to the module, not public. I made it private because it's only used from `runtime.rs`.

Reply:
Err, of course. My mistake.